server/utils/users/homePageDashboard.ts

```diff
@@ -90,5 +98,9 @@ export const sectionsForUser = (user: UserDetails): Array<ServiceSection> => {
     items.push(sections.reports)
   }
 
+  if (hasRole(user, 'role_admin')) {
```

Comment:
@pezholio I'm not sure if this is correct but it is the role that made the most sense

Reply:
We're letting `workflow_manager` roles edit users too 👍